hibernate-community-dialects/src/main/java/org/hibernate/community/dialect/DB2LegacySqlAstTranslator.java

```diff
@@ -429,11 +419,6 @@ protected String getFromDualForSelectOnly() {
 		return getFromDual();
 	}
 
-	@Override
-	protected void visitReturningColumns(MutationStatement mutationStatement) {
```

Comment:
I guess the removal of this was a victim of the renaming of `InsertStatement`? This should be aligned with `DB2SqlAstTranslator`.